odoo_backup_sh/__manifest__.py

```diff
@@ -8,9 +9,9 @@
     "images": ['images/odoo-backup.sh.jpg'],
     "version": "12.0.1.0.0",
 
-    "author": "IT-Projects LLC, Stanislav Krotov",
+    "author": "IT-Projects LLC",
     "support": "apps@it-projects.info",
-    "website": "https://apps.odoo.com/apps/modules/12.0/odoo_backup_sh/",
+    "website": "https://www.it-projects.info/",
```

Comment:
It was correct link. The link will work once we publish the module

Reply:
Returned back